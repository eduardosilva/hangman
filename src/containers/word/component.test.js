import React from 'react'
import renderer from 'react-test-renderer'
import Word from './component'

it('should render div', () => {
  const letter = renderer
    .create(<Word />)
    .toJSON()
  expect(letter).toMatchSnapshot()
})
