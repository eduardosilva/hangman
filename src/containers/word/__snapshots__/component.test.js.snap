// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render div 1`] = `
<div
  className={undefined}
/>
`;
